app/containers/SendTips/SendTipsForm.jsx

```diff
@@ -324,10 +328,27 @@ FormClone = connect(
     }
 
     const getInitialWallet = () => {
+      // there is saved data for send tips
+      const isPreselectedInWalletsArray =
+        tipsPreselect &&
+        wallets.find(wallet => wallet.name === tipsPreselect[WALLET_FIELD]);
+
+      if (isPreselectedInWalletsArray)
+        return Object.values(WALLETS).find(
+          wallet => wallet.name === tipsPreselect[WALLET_FIELD],
+        );
+
+      // there is no saved data for send tips
```

Comment:
wallets[0], wallets[1], wallets[2] are magic values that require guessing what these indexes mean.

Reply:
fixed